### ISSUE TITLE
This machine does not exist yet or this machine is incapable of having screenshots in its state.

### ISSUE THREAD

**Issue by luisdanielvc** [rainforestapp/tester-chrome-extension#369]
I don't understand why we have to struggle everyday with this, i'm always with fear when i get a long job because this error appears and there's nothing else to do more than refreshing and starting again.

**Comment by maxx88** [comment]
Yes, it's really annoying
![m](https://cloud.githubusercontent.com/assets/20210918/17323692/1d6b1496-58b4-11e6-8f21-98dcd8cd069d.jpg)

**Comment by luisdanielvc** [comment]
![zzzzzzzzzzz](https://cloud.githubusercontent.com/assets/16393092/17336796/5e66c7ba-58a5-11e6-9cfd-6b5219b8a759.PNG)
Beautiful, a 38 step job lost

**Comment by zfonck123** [comment]
i hate it,  always time get this error, need refresh and start again :(

**Comment by MarinaCec** [comment]
very annoying, it's the worst error on the new interface

**Comment by bradpittvn01** [comment]
I just got this today. I got a Job 38 Steps. I have done 35 Steps, step by step, it took me 30 minutes.
Then at Step 35, this error happened ......

30 minutes for nothing. Then no way, had to Refresh Job, and data was existing, could only complete job at Step 11! And because of data was existing, some screen-shorts was different with expected screen-short. Eventually this job was rejected.

40 minutes to do this Job, completed Step by Step honestly and carefully, then eventually .....

Really annoying and unfair.
